x-pack/test/functional/services/observability/alerts.ts

```diff
@@ -16,12 +17,18 @@ const DATE_WITH_DATA = {
 };
 
 const ALERTS_FLYOUT_SELECTOR = 'alertsFlyout';
+const ALERTS_TABLE_CONTAINER_SELECTOR = 'events-viewer-panel';
+
+const ACTION_COLUMN_INDEX = 1;
+
+type WorkflowStatus = 'open' | 'acknowledged' | 'closed';
```

Comment:
Just out of curiosity, what is the risk of pulling this from the production code base? 
Is it more from the perspective of blackbox testing or risk of coupling the code bases from a tooling perspective?

Reply:
The tooling can be problematic as well (TS performance in the `test` directories is terrible already), but the former point weighs heavier for me. These tests are supposed to be from the user's perspective and would therefore be aligned more with the UI concepts than the technical concepts. One example for that "looks the same, but actually isn't" is the fact that I needed to address the menu item via the verb "close" rather than the state "closed". So there is a mapping from action to state, but it's not identical.